lib/containerize_task.rb

```diff
@@ -43,4 +44,16 @@ def write_readme_file(dir)
       File.join(dir, "README.md")
     )
   end
+
+  private
+
+  def copy_workload_setup_files(description, workloads, services, path)
```

Comment:
Wouldn't it be nicer to make this somehow explicit? E.g. by encapsulating what is available to the template in a separate class?